myosuite/envs/myo/myoedits/__init__.py

```diff
@@ -0,0 +1,110 @@
+"""
+Copyright (c) 2024 MyoSuite
+Authors :: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano (caggiano@gmail.com), James Heald (jamesbheald@gmail.com)
+Source :: https://github.com/MyoHub/myosuite
+License :: Under Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+"""
+
+from myosuite.utils import gym; register=gym.register
+from myosuite.envs.myo.myobase import register_env_with_variants
+
+import os
+import numpy as np
+
+import mujoco
+
+curr_dir = os.path.dirname(os.path.abspath(__file__))
+
+# Arm Reaching ==============================
+def edit_fn_arm_reaching(spec: mujoco.MjSpec) -> None:
+
+	# Get the positions of each body of each digit as well as the properties of the IFtip site
+	root_list = ['firstmc', 'secondmc', 'thirdmc', 'fourthmc', 'fifthmc']
+	body_positions = {}
+	IFtip_site = {}
+	for root in root_list:
+		body_positions[root] = []
+		body = spec.body(root)
+		child_body = body.first_body()
+		while child_body is not None:
+			body_positions[root].append(child_body.pos.copy())
+			if child_body.name == 'distph2':
+				for site in child_body.sites:
+					if site.name == 'IFtip':
+						IFtip_site['name'] = site.name
+						IFtip_site['size'] = site.size.copy()
+						IFtip_site['pos'] = site.pos.copy()
+						IFtip_site['rgba'] = site.rgba.copy()
+			child_body = child_body.first_body()
+
+	# Remove the fingers
+	spec.detach_body(spec.body('proximal_thumb'))
```

Comment:
do those need to be harcorded or can they be extracted from the previous parent-child relationship?